### ISSUE TITLE
User can perform all functions of app using keyboard only.

### ISSUE THREAD

**Issue by brucestull** [brucestull/technology-and-perceptibility#48]
User needs to be able to navigate and perform all application functions using keyboard only.

This is important for accessibility.

**Comment by brucestull** [comment]
There is a bug:

1. "Skip to main content" link is used.
2. Use "View details of this item".
3. Use "Edit this item".
4. <Tab> selects the "Home" link in footer rather than first element in editing fieldset.

**Comment by brucestull** [comment]
Bug noted:
https://github.com/brucestull/technology-and-perceptibility/issues/83#issuecomment-1097459825

**Comment by brucestull** [comment]
Should be able to use [JavaScript window.location](https://www.w3schools.com/js/js_window_location.asp) to help fix this bug.
